stl/inc/strstream

```diff
@@ -242,7 +242,7 @@ protected:
             || (_Which & ios_base::out && !pptr())) { // N4727 [depr.strstreambuf.virtuals]/14:
                                                       // For a sequence to be positioned, if its next pointer
                                                       // is a null pointer, the positioning operation fails.
-            return pos_type(off_type(-1));
+            return {off_type(-1)};

```

Comment:
I suspect we can just return `-1LL` (ditto for other `pos_type(off_type(-1))`). IIUC both `pos_type` and `off_type` here are `long long`.